_includes/head.html

```diff
@@ -7,11 +7,13 @@
   <title>{% if page.title %}{{ page.title }}{% else %}{{ site.title }}{% endif %} | Demonstrate. Test. Approve.</title>
   <meta name="description" content="{% if page.excerpt %}{{ page.excerpt | strip_html | strip_newlines | truncate: 160 }}{% else %}{{ site.description }}{% endif %}">
 
-  <link rel="stylesheet" href="/css/probo_docs.css">
-
-  <link rel="icon" type="image/png" href="images/probo-sphere.png" />
   <link rel="canonical" href="{{ page.url | replace:'index.html','' | prepend: site.baseurl | prepend: site.url }}">
+
+  <link rel="stylesheet" href="/css/probo_docs.css">
   <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css" rel="stylesheet">
   <link href="https://cdn.rawgit.com/ProboCI/probo-styleguide/master/dist/probo.css" rel="stylesheet">
 
+  <!-- inject:html -->
+  <link rel="import" href="/images/favicons/icon.html">
```

Comment:
@tortillaj  gulp-inject's default behavior imports the `icon.html` file rather than including its contents a la `{% include /images/favicons/icons.html %}`